packages/common/sampleAddresses.ts

```diff
@@ -143,11 +143,11 @@ export const sampleAddresses: Record<ImplementedState, AddressData[]> = {
     state: 'Kansas',
   }],
   'Wisconsin': [{
-    address: '756 N Milwaukee St, Milwaukee, WI 53202',
-    city: 'Milwaukee',
+    address: '7421 W National Ave, West Allis, WI 53214',
+    city: 'West Allis',
     county: 'Milwaukee County',
-    postcode: '53202',
-    street: '756 N Milwaukee St',
+    postcode: '53214',
+    street: '7421 W National Ave',
     state: 'Wisconsin',
```

Comment:
The sample state wasn't returning any data from the API, I've checked this on `election-officials-data@2020-10-03` (before we made any changes for WI) and this address has `null` as its county.